src/posts/2020-07-27-unsucking-your-project-onboarding.md

```diff
@@ -0,0 +1,789 @@
+---
+path: /blog/unsucking-your-project-onboarding
+title: "Unsucking your project onboarding: bin/ scripts"
+date: 2020-07-27
+author: fernando-mendes
+tags:
+  - development
+  - devops
+intro: >
+  So you have a new developer joining your project. And, as usual, everything
+  changed in Javascript landscape since 37s ago. Now they have to install the
+  backend, frontend, all the tooling, etc etc etc. Stop this madness. The
+  programming languages and frameworks should be accessories to what you're doing.
+  They're tools you use and making use of them should be as simple as possible. At
+  Subvisual we have a system to handle this.
```

Comment:
This intro is wayyyy too long
not just for visual reasons in our blog post list, but for SEO reasons as well. it will end up being cut down to the first ~140 chars, so you may want to rework this

Reply:
tried, but not easy to do on build time.
currently enforcement is done in code reviews only

this is not a hard limit, since 140 chars is not a strict limit. Just a general commendation, since it's usually the maximum size you'll see in a lot of smaller devices or contexts